### ISSUE TITLE
Tracker: Hardware specific mods

### ISSUE THREAD

**Issue by trytomakeyouprivate** [iaacornus/Fedora-OSTree-Setup#51]
- [ ] AMD improvements
- [ ] Intel improvements
- [ ] Asahi M1 ?
- [ ] Arm ?
- [ ] Tuxedo Computer repository
- [ ] Thinkfan
- [ ] Framework improvements
- [ ] ~~Libfprint?~~

**Comment by trytomakeyouprivate** [comment]
The tuxedo repo could be added that way:

```
sudo wget https://rpm.tuxedocomputers.com/opensuse/15.4/repo-tuxedo-computers.repo -P /etc/yum.repos.d/Tuxedo.repo
```

The Tuxdedo Control center has nice features like a graphical hardware and energy manager too, so may be usable for any Laptop with similar hardware, maybe with a few tweaks. 

They include the packages packages:
- Tuxedofancontrol
- Tuxedo control center
- displaylink (the proprietary protocol?)
- anydesk
- rocketchat
- Zoiper

We should make sure it is chosen with lowest priority, as it may supply packages that are available in official repos.

**Comment by iaacornus** [comment]
`libfrint` as well as `fprintd` and others are already included by default

**Comment by iaacornus** [comment]
this should be included in #11 , im merging the two and close this

**Comment by iaacornus** [comment]
i did not included asahi m1 since it is quite out of scope, fedora silverblue, as far as i know doesnt run on it yet.
